Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/IndirectReplaceFitResult.py

```diff
@@ -62,14 +62,28 @@ def get_x_insertion_index(input_workspace, single_fit_workspace):
     return get_bin_index_of_value(input_workspace, bin_value)
 
 
+def get_indices_of_equivalent_labels(input_workspace, destination_workspace):
+    input_labels = input_workspace.getAxis(1).extractValues()
+    labels = destination_workspace.getAxis(1).extractValues()
+    return [index for index, label in enumerate(labels) if label in input_labels]
+
+
+def fit_parameter_missing(single_fit_workspace, destination_workspace, exclude_parameters):
+    single_parameters = single_fit_workspace.getAxis(1).extractValues()
+    destination_parameters = destination_workspace.getAxis(1).extractValues()
+    return any([True for parameter in single_parameters
```

Comment:
```suggestion
    return any([parameter not in destination_parameters and parameter not in exclude_parameters for parameter in single_parameters])
```
I feel this might be a clearer statement as you don't have the hard-coded `True`